components/browser/engine-gecko/src/main/java/mozilla/components/browser/engine/gecko/prompt/GeckoPromptDelegate.kt

```diff
@@ -224,6 +227,39 @@ internal class GeckoPromptDelegate(private val geckoEngineSession: GeckoEngineSe
         return geckoResult
     }
 
+    override fun onAddressSelect(
+        session: GeckoSession,
+        request: AutocompleteRequest<Autocomplete.AddressSelectOption>
+    ): GeckoResult<PromptResponse>? {
```

Comment:
```suggestion
    ): GeckoResult<PromptResponse> {
```
I think Android Studio also says the `?` is unnecessary since we always return a result.